examples/takeoff_and_land.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
+
+from dronecode_sdk import (AsyncPluginManager,
```

Comment:
I actually added a `connect` function in [__init__.py](https://github.com/Dronecode/DronecodeSDK-Python/blob/master/dronecode_sdk/__init__.py) which setups the default plugins and connects to the backend.

You can use it like this:
```Python
from dronecode_sdk import connect as dronecode_sdk_connect

copter = dronecode_sdk_connect(host="127.0.0.1")

# And inside your funcions you can access the plugins as a simple property, e.g. action:
await copter.action.arm()
```